client/src/Commons/FormInput.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
```

Comment:
this component already exists in the master branch. Check if you merged the master correctly.